src/components/Home/NoticeFunSystem/NoticeFunSystem.tsx

```diff
@@ -0,0 +1,53 @@
+import { useState } from 'react';
+import * as styles from './NoticeFunSystem.style';
+import RightArrow from '@icons/icon/Arrow/RightSmallArrow.svg';
+import Image from 'next/image';
+
+interface SummaryText{
+  textContent : string;
+  date : string;
+}
+
+export interface NoticeFunSystemProps{
+  title : string;
+  summaries : SummaryText[];
+}
+
+
+const NoticeFunSystem= (props:NoticeFunSystemProps) => {
+
+  return (
+      <styles.SummaryBox>
+        <styles.TopBox>
+          <styles.Title>
+            {props.title}
+          </styles.Title>
+          <styles.MoreButton>
+            더보기
+            <Image src ={RightArrow} alt = "right arrow"/>
+          </styles.MoreButton>
+        </styles.TopBox >
+        
+        <styles.TextBox>
+          <styles.Line top="55px"/>
+          <styles.Line top="112px"/>
+          {props.summaries.map((summary, index) => (
+            <div key={index}>
+              <styles.TextLine top={`${16 + index * 57}px`}>
+                <styles.TextSummary>
+                  {summary.textContent}
+                </styles.TextSummary>
+                <styles.Date>
+                  {summary.date}
+                </styles.Date>
+              </styles.TextLine>
+              
+            </div>
```

Comment:
div로 묶은 이유가 따로 없다면 key는 내부에 넣어도 될 것 같아요!
```suggestion
              <styles.TextLine top={`${16 + index * 57}px`} key={index}>
                <styles.TextSummary>
                  {summary.textContent}
                </styles.TextSummary>
                <styles.Date>
                  {summary.date}
                </styles.Date>
              </styles.TextLine>
```